CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## X.X.X
-* Mitigated an issue where some crash reports are not automatically sent when you `enableCrashReporting` from config.
-* Removed all reference to `runZonedGuarded`. Dart and async errors are caught using `PlatformDispatcher.instance.onError` if crashReporting is enabled.
+* Resolved an issue where certain crash reports were not automatically sent.
+* Removed all reference to `runZonedGuarded`.
```

Comment:
yea this was the detail I had in mind :D they dont have to know we removed mentions of this. 

But you fixed 2 things:
1 automatic crash reporting did not include flutter framework level errors with the new config option we introduced but u fixed it
2. we were not capturing async dart errors but now we do

Reply:
Done